packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/useManageContestForm.ts

```diff
@@ -26,9 +27,21 @@ const useManageContestForm = ({
         return;
       }
 
+      const contestLengthInSeconds = moment(
+        contestData?.contests?.[0]?.end_time,
```

Comment:
what if its not defined?

Reply:
technically it is NOT possible to be at this view (editing contest) and do not have `end_time` and `start_time` of the contest. There is bunch of optional chaining because of typescript though